tests/python/pytorch/graphbolt/impl/test_torch_based_feature_store.py

```diff
@@ -57,13 +61,24 @@ def test_torch_based_feature(in_memory):
             feature_b.read(), torch.tensor([[[1, 2], [3, 4]], [[1, 2], [3, 4]]])
         )
 
+#         # Test get the size of the entire feature.
```

Comment:
Remove if we don't need this, never just comment out unused code.

Reply:
OK. This code I will modify later to test the size API.